src/Fetcher/TululaFetcher.php

```diff
@@ -0,0 +1,241 @@
+                $endDate = \DateTimeImmutable::createFromFormat('Y-m-d', $rawConference['dateEnd']);
+                $conference->setEndAt($endDate);
+            }
+
+            if ($rawConference['cfpUrl']) {
+                $conference->setCfpUrl($rawConference['cfpUrl']);
+            }
+
+            if ($rawConference['cfpDateEnd']) {
+                $cfpEndAt = \DateTimeImmutable::createFromFormat('Y-m-d', $rawConference['cfpDateEnd']);
+                $conference->setCfpEndAt($cfpEndAt);
+            }
+            $conferences[] = $conference;
+        }
+
+        return $conferences;
+    }
+
+    private function queryTululaApi()
+    {
+        try {
+            $response = $this->client->request('POST', $this->getUrl(), [
+                'headers' => [
+                    'Content-Type' => 'application/json',
+                ],
+                'json' => [
+                    'variables' => [
+                        'filter' => [
+                            'cfpFrom' => date('Y-m-d'),
+                            'cfpIsActive' => true,
+                        ],
+                    ],
+                    'query' => <<<QUERY
+                        query QueryEventsSearch(\$filter: EventSearchFilter) {
+                            events: eventsSearch(filter: \$filter) {
+                                events {
+                                ...EventData
+                                }
+                            }
+                        }
+                        fragment EventData on Event {
+                            name
+                            url
+                            dateStart
+                            dateEnd
+                            cfpDateEnd
+                            cfpUrl
+                            isOnline
+                            slug
+                            venue {
+                                ...VenueData
+                            }
+                            tags {
+                                ...TagData
+                            }
+                        }
+                        fragment VenueData on Venue {
+                            countryCode
+                            state
+                            city
+                        }
+                        fragment TagData on Tag {
+                            name
+                        }
+                    QUERY
+                ],
+            ]);
+        } catch (TransportExceptionInterface $exception) {
+            $this->logger->error('HttpClient Transport Exception', [
+                'url' => $this->getUrl(),
+                'source' => self::SOURCE,
+                'exception' => $exception->getMessage(),
+            ]);
+
+            return;
+        }
+
+        if (404 === $response->getStatusCode()) {
+            $this->logger->error('Source URL returns 404', ['url' => $this->getUrl(), 'source' => self::SOURCE]);
+
+            return;
+        }
+
```

Comment:
What about all other status code ? like 5XX?
May be it is already handler at another level but I did not see it in the diff